app/pulp/app/serializers/user.py

```diff
@@ -14,6 +14,9 @@ class UserSerializer(ModelSerializer):
                   "them."
     )
 
+    password = serializers.CharField(write_only=True, help_text="Password", min_length=5,
```

Comment:
I think the min and max lengths are new, how did you choose these numbers?

Enforcing this might be fine, but this would need to be a release note. We don't really need to worry about backwards compatibility here because we won't be able to migrate the old passwords anyway.